### ISSUE TITLE
ui-svg (imported as 'zeroLength') was not found in '@nativescript/core' 

### ISSUE THREAD

**Issue by serrg1994** [nativescript-community/ui-canvas#42]
I have installed 0.1.2 version for ui-svg library and nativescript core 8.4.4.

Application is compiling and rendering SVGs, I don't see any problems with it, but during compilation I always see the next Warning

`WARNING in ./node_modules/@nativescript-community/ui-svg/index.common.js 42:19-29
export 'zeroLength' (imported as 'zeroLength') was not found in '@nativescript/core' (possible exports: AbsoluteLayout, AccessibilityEvents, AccessibilityLiveRegion, AccessibilityRole, AccessibilityState, AccessibilityTrait, ActionBar, ActionItem, ActionItems, ActivityIndicator, AndroidApplication, Animation, AnimationCurve, Application, ApplicationSettings, Background, Binding, Builder, Button, CSSHelper, CSSType, CSSUtils, ChangeType, CoercibleProperty, Color, Connectivity, ContainerView, ContentView, ControlStateChangeListener, CoreTypes, CssAnimationParser, CssAnimationProperty, CssProperty, CustomLayoutView, DatePicker, Device, DialogStrings, Dialogs, DockLayout, EditableTextBase, Enums, File, FileSystemEntity, FlexboxLayout, Folder, Font, FontScaleCategory, FontStyle, FontVariationSettings, FontWeight, FormattedString, Frame, GestureEvents, GestureStateTypes, GestureTypes, GesturesObserver, GridLayout, GridUnitType, HtmlView, Http, IOSHelper, Image, ImageAsset, ImageCache, ImageSource, InheritedCssProperty, InheritedProperty, ItemSpec, KeyframeAnimation, KeyframeAnimationInfo, KeyframeDeclaration, KeyframeInfo, Label, LayoutBase, Length, ListPicker, ListView, ModuleNameResolver, NavigationButton, Observable, ObservableArray, Page, PageBase, ParserEvent, ParserEventType, PercentLength, Placeholder, Progress, Property, ProxyViewContainer, PseudoClassHandler, Repeater, RootLayout, Screen, ScrollView, SearchBar, SegmentedBar, SegmentedBarItem, ShorthandProperty, Slider, Span, StackLayout, Style, SwipeDirection, Switch, TabView, TabViewItem, TextBase, TextField, TextView, TimePicker, TouchAction, TouchManager, Trace, Transition, Utils, View, ViewBase, ViewHelper, VirtualArray, WebView, WrapLayout, WrappedValue, XmlParser, _setResolver, action, addTaggedAdditionalCSS, addWeakEventListener, alert, androidDynamicElevationOffsetProperty, androidElevationProperty, animationTimingFunctionConverter, autocapitalizationTypeProperty, autocorrectProperty, autofillTypeProperty, backgroundColorProperty, backgroundImageProperty, backgroundInternalProperty, backgroundPositionProperty, backgroundRepeatProperty, backgroundSizeProperty, booleanConverter, borderBottomColorProperty, borderBottomLeftRadiusProperty, borderBottomRightRadiusProperty, borderBottomWidthProperty, borderLeftColorProperty, borderLeftWidthProperty, borderRightColorProperty, borderRightWidthProperty, borderTopColorProperty, borderTopLeftRadiusProperty, borderTopRightRadiusProperty, borderTopWidthProperty, capitalizationType, clipPathProperty, colorProperty, confirm, dismissKeyboardTimeoutId, eachDescendant, editableProperty, encoding, fontFamilyProperty, fontInternalProperty, fontScaleProperty, fontSizeProperty, fontStyleProperty, fontVariationSettingsProperty, fontWeightProperty, fromObject, fromObjectRecursive, getAncestor, getCurrentPage, getFileAccess, getRootLayout, getTransformedText, getViewById, heightProperty, hintProperty, horizontalAlignmentProperty, iOSApplication, inputType, isAndroid, isIOS, isUserInteractionEnabledProperty, keyboardTypeProperty, knownFolders, letterSpacingProperty, lineHeightProperty, login, makeParser, makeValidator, marginBottomProperty, marginLeftProperty, marginRightProperty, marginTopProperty, maxLengthProperty, minHeightProperty, minWidthProperty, opacityProperty, paddingBottomProperty, paddingLeftProperty, paddingRightProperty, paddingTopProperty, parseCSSShadow, parseKeyframeDeclarations, path, perspectiveProperty, placeholderColorProperty, platformNames, profile, profilingDisable, profilingDumpProfiles, profilingEnable, profilingIsRunning, profilingResetProfiles, profilingStart, profilingStartCPU, profilingStop, profilingStopCPU, profilingTime, profilingUptime, prompt, removeTaggedAdditionalCSS, removeWeakEventListener, resolveFileNameFromUrl, returnKeyTypeProperty, rotateProperty, rotateXProperty, rotateYProperty, sanitizeModuleName, scaleXProperty, scaleYProperty, setActivityCallbacks, textAlignmentProperty, textDecorationProperty, textShadowProperty, textTransformProperty, timeConverter, transformConverter, translateXProperty, translateYProperty, unsetValue, updateTextTriggerProperty, verticalAlignmentProperty, visibilityProperty, whiteSpaceProperty, widthProperty, zIndexProperty)
 @ ./node_modules/@nativescript-community/ui-svg/index.android.js 2:0-105 3:0-43 3:0-43 48:25-32 174:86-104 323:29-40 334:5-26
 @ ./node_modules/@nativescript-community/ui-svg/vue/index.js 1:0-45 4:47-56 5:45-52 6:42-45
 @ ./app/main.js 24:0-59 25:8-17`

My dependencies 

` "dependencies": {
    "@nativescript-community/ui-material-bottomsheet": "^7.0.34",
    "@nativescript-community/ui-material-button": "7.0.34",
    "@nativescript-community/ui-material-ripple": "7.0.34",
    "@nativescript-community/ui-svg": "^0.1.2",
    "@nativescript/camera": "^5.0.15",
    "@nativescript/core": "8.4.4",
    "@nativescript/firebase": "^11.1.3",
    "@nstudio/nativescript-pulltorefresh": "^3.0.2",
    "@triniwiz/nativescript-socketio": "^4.0.3",
    "axios": "^0.18.0",
    "nativescript-permissions": "^1.3.6",
    "nativescript-vue": "^2.9.3",
    "vuex": "^3.0.1"
  },
  "devDependencies": {
    "@babel/core": "^7.8.3",
    "@babel/preset-env": "^7.8.3",
    "@nativescript/android": "~8.4.0",
    "@nativescript/ios": "8.4.0",
    "@nativescript/webpack": "~5.0.0",
    "babel-loader": "^8.0.6",
    "babel-traverse": "6.26.0",
    "babel-types": "6.26.0",
    "babylon": "6.18.0",
    "lazy": "1.0.11",
    "nativescript": "^8.4.0",
    "nativescript-vue-template-compiler": "^2.0.0",
    "sass": "~1.49.9",
    "vue-loader": "^15.8.3"
  },`

**Comment by farfromrefug** [comment]
@serrg1994 @vallemar sorry about this. Will try to fix it today

**Comment by farfromrefug** [comment]
@serrg1994 @vallemar i tried but i cant reproduce the issue :s not. Could be the typescript versoin. Which one are you using?

**Comment by vallemar** [comment]
@farfromrefug  I have this version: `"typescript": "~4.8.4",`

**Comment by farfromrefug** [comment]
@vallemar using the same one :s

**Comment by vallemar** [comment]
@farfromrefug Until I register the plugin in vue, I don't get the error. It's not worth just adding it as a dependency. I don't know if this is what could be happening to you

**Comment by farfromrefug** [comment]
@vallemar i tried with my demo vue app and have no warning :s . plus the warning is wrong as it is exported

**Comment by vallemar** [comment]
@farfromrefug Yes, I have been keeping an eye on it and I have seen that it is exported. I don't know why it is giving this error

**Comment by serrg1994** [comment]
Which nativescript core version are you testing?

**Comment by farfromrefug** [comment]
@serrg1994 8.4.4
